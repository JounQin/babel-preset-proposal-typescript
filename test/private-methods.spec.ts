import { Counter, Resource } from './private-methods'
import { execute } from './helpers'

const proposal = 'private-methods'

test(proposal, () => {
  expect(() => execute(proposal)).toThrowErrorMatchingSnapshot()
  const counter = new Counter()
  expect(counter.getCount()).toBe(0)
  expect(counter.increment().getCount()).toBe(1)
  expect(new Resource('test').status).toBe('test_loaded')
  expect(new Resource('test2').status).toBe('test2_loaded')
})
