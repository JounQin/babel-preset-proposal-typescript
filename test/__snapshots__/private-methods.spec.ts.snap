// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`private-methods 1`] = `"Unexpected token ';'"`;
